src/java.desktop/share/classes/sun/swing/FilePane.java

```diff
@@ -1123,11 +1124,15 @@ public Component getTableCellEditorComponent(JTable table, Object value,
     class DetailsTableCellRenderer extends DefaultTableCellRenderer {
         JFileChooser chooser;
         DateFormat df;
+        final static double baseFileSize = 1000.0;
```

Comment:
let's use blessed modifiers order: `static final`

Reply:
Thanks for catching it, I missed it.

Reply:
@turbanoff @aivanov-jdk Modifier's order updated.